src/core/lib/iomgr/tcp_server_windows.cc

```diff
@@ -95,16 +99,17 @@ struct grpc_tcp_server {
   /* shutdown callback */
   grpc_closure* shutdown_complete;
 
-  grpc_channel_args* channel_args;
+  grpc_tcp_generic_options options;
```

Comment:
Doesn't look like `grpc_tcp_create()` in the Windows iomgr impl actually uses this, so we can probably just remove this field.

Reply:
Removed it in the latest commit.